OpenEdXMobile/src/main/java/org/edx/mobile/googlecast/CastOptionsProvider.java

```diff
@@ -0,0 +1,58 @@
+package org.edx.mobile.googlecast;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.cast.CastMediaControlIntent;
+import com.google.android.gms.cast.MediaMetadata;
+import com.google.android.gms.cast.framework.CastOptions;
+import com.google.android.gms.cast.framework.OptionsProvider;
+import com.google.android.gms.cast.framework.SessionProvider;
+import com.google.android.gms.cast.framework.media.CastMediaOptions;
+import com.google.android.gms.cast.framework.media.ImageHints;
+import com.google.android.gms.cast.framework.media.ImagePicker;
+import com.google.android.gms.cast.framework.media.NotificationOptions;
+import com.google.android.gms.common.images.WebImage;
+
+import java.util.List;
+
+/**
+ * *** DO NOT DELETE THIS FILE ***
+ * App must implement the OptionsProvider interface to supply options needed to initialize the CastContext singleton.
+ * Ref https://developers.google.com/cast/docs/android_sender/integrate#initialize_the_cast_context
+ */
+public class CastOptionsProvider implements OptionsProvider {
```

Comment:
better to write some Gradle script to verify such file existence in unused/dynamically used file scenarios.